src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -156,5 +165,16 @@ export default class SimidMapCreative extends BaseSimidCreative {
       map: map,
       title: 'Current Position'
     });
+    let currentUserSession_ = this.newUserSession_;
```

Comment:
are you saving it here b/c of the complications with 'this'? if so try using the arrow functions. More on that below

Reply:
got rid of it